test/peripheral/522_borrowProxy_signatures.ts

```diff
@@ -112,36 +114,18 @@ contract('BorrowProxy - Signatures', async (accounts) => {
           assert.equal(await dai.balanceOf(user2), oneToken.toString())
         })
 
-        it("doesn't borrow dai if limit exceeded", async () => {
-          await expectRevert(
-            proxy.borrowDaiForMaximumFYDaiWithSignature(
-              pool.address,
-              WETH,
-              maturity1,
-              user2,
-              fyDaiTokens1,
-              daiTokens1,
-              controllerSig,
-              {
-                from: user1,
-              }
-            ),
-            'BorrowProxy: Too much fyDai required'
-          )
-        })
-
```

Comment:
Why is this test removed? Should we keep it?

Reply:
Tested in 521, line 185.